platforms/Cross/vm/sqAtomicOps.h

```diff
@@ -164,9 +169,39 @@ AtomicGet(uint64_t *target)
 							: "m" (lo32(value)), "m" (hi32(value)) \
 							: "memory", "eax", "ebx", "ecx", "edx", "cc")
 #  endif /* __SSE2__ */
-#elif defined(_MSC_VER) && (defined (_M_AMD64) || defined(_MX64) || defined(_M_IA64))
```

Comment:
Hi Tobias,
why do you remove those lines? I don't see them elsewhere.
64bits ops are atomics on win64, so I think they are OK.